app/locustio/jira/http_actions.py

```diff
@@ -47,14 +48,13 @@ def login_and_view_dashboard(locust):
                       catch_response=True)
     # Assertions
     token = fetch_by_re(params.atl_token_pattern, content)
-    if not token:
-        logger.info(f'{params.action_name}: {content}')
-        raise ResponseError('Atlassian token not found in login requests')
+    if not (f'title="loggedInUser" value="{user[0]}">' in content):
+        logger.error(f'User {user[0]} authentication failed')
```

Comment:
need to add `content` like:
`logger.error(f'User {user[0]} authentication failed: {content}')`